src/worker.py

```
@@ -871,6 +1065,228 @@ async def api_login(req, env):
+        ).bind(row.user_id).run()
+        await env.DB.prepare(
+            "DELETE FROM email_verification_tokens WHERE id=?"
+        ).bind(row.id).run()
+    except Exception as exc:
+        await capture_exception(exc, req, env, "api_verify_email.update")
+        return err("Verification failed — please try again", 500)
+
+    return ok(None, "Email verified successfully. You can now sign in.")
+
+
+async def api_resend_verification(req, env):
```

Ruff (0.15.15)

[warning] Too many return statements (7 > 6)

(PLR0911)